spotify_project/libraries/models.py

```diff
@@ -27,3 +28,11 @@ def top_artists(self):
         list_tuples.sort(key=itemgetter(1), reverse=True)
         result = dict(list_tuples[:10])
         return result
+
+    @property
+    def total_duration(self):
+        result = 0
+        for track in self.tracks.all():
+            result += int(track.duration_ms)
+        result = result // 1000000 * 1000000
```

Comment:
What's going on here? You should Google `Decimal`. It's likely what your want both here and we the field in the ORM